Source/Core/VideoCommon/TextureInfo.cpp

```diff
@@ -86,6 +86,8 @@ TextureInfo::TextureInfo(const u8* ptr, const u8* tlut_ptr, u32 address,
 
     // load mips - TODO: Loading mipmaps from tmem is untested!
```

Comment:
Shouldn't this comment be modified?

Reply:
I dunno, I didn't put there.

Reply:
I guess this counts as testing it, since the fact this fixes the issue proves that the game is loading mipmaps from tmem.

Reply:
Yeah, this was an old comment from the previous code, no idea why it was there.  I just left it in when I ported it over.